doc/general/cash-register-integration/cash-register-integration-failure-scenarios.md

```diff
@@ -0,0 +1,32 @@
+### Failure Scenarios
+This Chapter describes the different scenarios of failure when using the fiskaltrust.Middleware and how to handle them.
+
+#### Failure of the fiskaltrust.Middleware
+If a cash register cannot communicate with the fiskaltrust.Middleware it is most likely due to a failure of the fiskaltrust Middleware. Such a failure means that the electronic recording system is not operational and there is no access to the appropriate journal.
+
+![](./images/07-no-middleware-connection.png)
```

Comment:
It's usually a best practice to add alt names to images for accessibility issues:
```suggestion
![no-middleware-connection](./images/07-no-middleware-connection.png)
```

Reply:
Also happens a few times below, maybe we can fix this (even if it already was like that before 😅)